plugins/kubernetes/app/models/kubernetes/release.rb

```diff
@@ -54,6 +53,10 @@ def url
       Rails.application.routes.url_helpers.project_kubernetes_release_url(project, self)
     end
 
+    def builds
+      Build.where(git_sha: git_sha)
```

Comment:
It's possible for more than one `Project` to point to the same repository. So I'd scope this by the project, just to be safe.

Reply:
builds are unique on the git-sha so that should be fine ... but still changed just to be safe ...